RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -3714,6 +3847,14 @@
 				minimumVersion = 9.0.0;
 			};
 		};
+		5759B336296DF65D002472D5 /* XCRemoteSwiftPackageReference "nimble" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/quick/nimble";
+			requirement = {
+				kind = upToNextMinorVersion;
```

Comment:
FFTI but I wonder if we could make this go up to the next major version, since it's only for tests and minors "shouldn't" break things. I'm ok leaving as is though.

Reply:
The next major doesn't support iOS 11.x unfortunately.

Reply:
Oh damn sorry I forgot to update this.
Either way Nimble 10.x only has a single release: 10.0.0.